packages/client/components/ActivityLibrary/ActivityCard.tsx

```diff
@@ -0,0 +1,48 @@
+import clsx from 'clsx'
+import React from 'react'
+import {MeetingTypeEnum} from '../../__generated__/ActivityLibraryQuery.graphql'
+
+interface CardTheme {
+  primary: string
+  secondary: string
+}
+
+export const MeetingThemes: Record<MeetingTypeEnum, CardTheme> = {
+  action: {primary: 'bg-aqua-400', secondary: 'bg-[#CBECF0]'},
+  poker: {primary: 'bg-tomato-500', secondary: 'bg-[#FFE2E0]'},
+  retrospective: {primary: 'bg-grape-500', secondary: 'bg-[#F2E1F7]'},
+  teamPrompt: {primary: 'bg-aqua-400', secondary: 'bg-[#CBECF0]'}
+}
+
+interface Props {
+  className?: string
+  type: MeetingTypeEnum
+  children: React.ReactNode
+}
+
+export const ActivityCard = (props: Props) => {
```

Comment:
This is non interactive activity card that could be reused on meeting dashboard later on

Reply:
+1 Is this pulled out specifically for reuse later on, or is the core intent clean code in this PR? It seems like there's a _lot_ more that we'd want to reuse for the meeting dashboard, so if this is intended for reuse, we should probably extract almost all of the `ActivityLibraryCard` - basically everything except the badges